docs/encyclopedia/temporal-nexus-overview.mdx

```diff
@@ -0,0 +1,118 @@
+---
+id: nexus-overview
+title: Temporal Nexus Overview
+sidebar_label: Overview
+description: Temporal Nexus is designed to connect durable executions across team and Namespace boundaries with an integrated Temporal SDK experience, promoting a modular architecture for sharing your team’s capabilities via a well-defined Nexus Service API contract.
+toc_max_heading_level: 4
+slug: /temporalnexus/overview
+keywords:
+  - modular architecture
+  - Namespace boundaries
+  - durable execution
+tags:
+  - distributed systems
+  - integration
+  - temporal
+---
+
```

Comment:
Most of this content here is the same content from the [Evaluate page](https://temporal-documentation-git-feature-landing-page-nexus.preview.thundergun.io/nexus). It's going to be repetitive and users will end up reading it twice. Ideally, users will start at the evaluate page then go to the encyclopedia next. Is there a way to sumarize the first four paragraphs. Otherwise, we could keep the first paragraph as the 2-liner intro about Nexus and jump into the Nexus RPC and continue from there.